projs/shadow/shadow-engine/core/src/core/ShadowApplication.cpp

```diff
@@ -31,54 +31,59 @@ namespace ShadowEngine {
 			for (size_t i = 0; i < argc; i++)
 			{
 				std::string param(argv[i]);
-				if(param == "-no-gui")
+				if(param == "-debug")
 				{
-					this->no_gui = true;
+					this->debug = true;
 				}
                 if(param == "-game")
                 {
                     this->game = argv[i+1];
                 }
 			}
 		}
+        SetConsoleOutputCP(CP_UTF8);
```

Comment:
Where in the name of the great lovable bejeebus did this come from? What's it do? Where are the comments? Where, oh merciful lord above, are the comments?